pennylane_lightning/lightning_gpu/_adjoint_jacobian.py

```diff
@@ -100,21 +189,43 @@ def calculate_jacobian(self, tape: QuantumTape):
         if empty_array:
             return np.array([], dtype=self.dtype)
 
-        processed_data = self._process_jacobian_tape(tape)
+        if self._use_mpi:
+            split_obs = False  # with MPI batched means compute Jacobian one observables at a time, no point splitting linear combinations
+        else:
+            split_obs = self._dp.getTotalDevices() if self._batch_obs else False
```

Comment:
Do we want raise errors when users turn on _batch_obs and mpi at the same time?

Reply:
It would be better to launch a warning than a raise. 
Are you agree with the following warning:
```python 
# Warning about performance with MPI and batch observation 
if self._use_mpi and self._batch_obs:
    warn("It is not possible to use LGPU with MPI and batch the observation. batch_obs will be deactivated",RuntimeWarning)
    self._batch_obs = False
```